src/App.js

```diff
@@ -1,42 +1,21 @@
-import "./App.css";
-import logo from "./logo.png";
+import {Routes, Route} from "react-router-dom"
+import Cart from "./pages/cart/Cart.jsx"
+import Wishlist from "./pages/wishlist/Wishlist.jsx"
+import Signup from "./pages/signup/Signup.jsx"
+import Login from "./pages/login/Login.jsx"
+import Products from "./pages/products/Products.jsx";
+import Home from "./pages/home/Home";
```

Comment:
Create index file in pages folder and export all the pages from the index file and then import all the pages from the single index file here.

Reply:
Have import/export all the components from index file, good pointer!